wordfreq/mecab.py

```diff
@@ -1,12 +1,61 @@
 from pkg_resources import resource_filename
 import MeCab
 import unicodedata
+import os
+
+
+# MeCab has fixed-sized buffers for many things, including the dictionary path
+MAX_PATH_LENGTH = 58
+
+
+def find_mecab_dictionary(names):
+    """
+    Find a MeCab dictionary with a given name. The dictionary has to be
+    installed separately -- see wordfreq's README for instructions.
+    """
+    suggested_pkg = names[0]
+    paths = [
+        os.path.expanduser('~/.local/lib/mecab/dic'),
+        '/var/lib/mecab/dic',
+        '/var/local/lib/mecab/dic',
+        '/usr/lib/mecab/dic',
+        '/usr/local/lib/mecab/dic',
+    ]
+    full_paths = [os.path.join(path, name) for path in paths for name in names]
```

Comment:
Would it be worth consolidating the `MAX_PATH_LENGTH` logic?  For example, compute `skipped_paths` and `checked_paths` up here, or something.